bootstrap/eks/config/rbac/leader_election_role_binding.yaml

```diff
@@ -8,5 +8,5 @@ roleRef:
   name: leader-election-role
 subjects:
 - kind: ServiceAccount
-  name: default
+  name: controller-manager
```

Comment:
I ran into this too, should it be a separate PR?

Reply:
Yes it should, i have created #2183

Reply:
Separate PR created and this has been removed from this change